statusreports/REPORT_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+# [Month Day Year] Week in Review 
+
+### Notable Pull Requests/Issues
+
+- Was something significant merged in the last week?  [Resolves #115](https://example.com/resolved/pull/request)
+- Was a PR submitted for a significant change? [In Progress #120](https://example.com/pull/request)
+- Is there new/dynamic conversation happening about a PR/issue that readers should look at? [Issue #100](https://example.com/open/issue)
+
+### Mailing List Discussions
+
+- Discussions happening that readers should look at (summarize and point to ML archives)
+
+### Community Updates
+
+- Is someone presenting a talk?
+- Noteworthy Kata news? (i.e. "Became an OCI member this week)
```

Comment:
Missing double-quote before the closing bracket?

Reply:
@jodh-intel fixed